utils/io/fileutils/files.go

```diff
@@ -366,17 +359,14 @@ func GetFileDetails(filePath string, includeChecksums bool) (details *FileDetail
 
 	file, err := os.Open(filePath)
 	defer func() {
-		e := file.Close()
-		if err == nil {
-			err = errorutils.CheckError(e)
-		}
+		err = errors.Join(err, errorutils.CheckError(file.Close()))
 	}()
-	if errorutils.CheckError(err) != nil {
-		return nil, err
+	if err != nil {
```

Comment:
whats the difference?

Reply:
It's just to be consistent with the rest of the code